pinot-query-runtime/src/main/java/org/apache/pinot/query/service/server/QueryServer.java

```diff
@@ -71,7 +88,17 @@ public void start() {
     LOGGER.info("Starting QueryServer");
     try {
       if (_server == null) {
-        _server = ServerBuilder.forPort(_port).addService(this).maxInboundMessageSize(MAX_INBOUND_MESSAGE_SIZE).build();
+        if (_tlsConfig == null) {
+          try {
+            _server =
+                ServerBuilder.forPort(_port).addService(this).maxInboundMessageSize(MAX_INBOUND_MESSAGE_SIZE).build();
```

Comment:
Why is try-catch required now? Since this line is the same as before.

Reply:
Done. Although I think try-catch should be there.